sdks/python/apache_beam/io/gcp/bigtableio_it_test.py

```diff
@@ -50,6 +50,7 @@
 
 
 @pytest.mark.uses_gcp_java_expansion_service
+@pytest.mark.uses_transform_service
```

Comment:
I may be missing something but how does this test suite make sure to build the necessary jars before running? For example in this test we need `:sdks:java:io:google-cloud-platform:expansion-service`